src/main/scala/firrtl/annotations/JsonProtocol.scala

```diff
@@ -77,7 +77,14 @@ object JsonProtocol {
   }.recoverWith { // If the input is a file, wrap in InvalidAnnotationFileException
     case e => in match {
       case FileInput(file) =>
-        Failure(new InvalidAnnotationFileException(file, e))
+        /* This occurs during scopt parsing, however, scopt will only print the
+         * first error message and does not properly deal with nested
+         * exceptions. Hence, the better processing below of wrapping the
+         * annotation with an [[InvalidAnnotationFileException]] is
+         * currently disabled for improved user understanding of the
+         * error. */
+        /* Failure(new InvalidAnnotationFileException(file, e)) */
+        Failure(e)
```

Comment:
So this means the JSON exception will still be reported, it just won't indicate what the Annotation filename is? Not ideal but I guess it could be worse. Would be nice to get this behavior back but I wouldn't block this PR over it

Reply:
I need to investigate this and get a small example provided... (I can't remember exactly.)